pom.xml

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.424</version>
+    <version>1.596.1</version>
```

Comment:
> Firstly, that is a rather obnoxious comment IMO

Sorry for that.

I referenced previous reviews, but I cannot get why this version is required.
It looks expected to know more details about previous reviews, and I'm afraid I'm not a good reviewer for this request.

Anyway, the policy for the targeting Jenkins version is up to the plugin maintainers, and we can wait for the review of them.